x-pack/plugins/enterprise_search/README.md

```diff
@@ -2,16 +2,28 @@
 
 ## Overview
 
-This plugin's goal is to provide a Kibana user interface to the Enterprise Search solution's products (App Search and Workplace Search). In it's current MVP state, the plugin provides the following with the goal of gathering user feedback and raising product awareness:
+This plugin provides beta Kibana user interfaces for managing the Enterprise Search solution and its products, App Search and Workplace Search.
 
-- **App Search:** A basic engines overview with links into the product.
-- **Workplace Search:** A simple app overview with basic statistics, links to the sources, users (if standard auth), and product settings.
+> :warning: The Kibana interface for Enterprise Search is a beta feature. It is subject to change and is not covered by the same level of support as generally available features. This interface will become the sole management panel for Enterprise Search with the 8.0 release. Until then, the standalone Enterprise Search UI remains available and supported.
+
+### App Search
+
+<img src="./public/applications/enterprise_search/assets/app_search.png" width="400" height="255">
+
+Add rich, relevant search to your apps and websites. https://www.elastic.co/app-search/
+
+### Workplace Search
+
+<img src="./public/applications/enterprise_search/assets/workplace_search.png" width="400" height="255">
+
+Unify all your team's content into a personalized search experience. https://www.elastic.co/workplace-search/
 
 ## Development
 
 1. When developing locally, Enterprise Search should be running locally alongside Kibana on `localhost:3002`.
 2. Update `config/kibana.dev.yml` with `enterpriseSearch.host: 'http://localhost:3002'`
-3. For faster QA/development, run Enterprise Search on [elasticsearch-native auth](https://www.elastic.co/guide/en/app-search/current/security-and-users.html#app-search-self-managed-security-and-user-management-elasticsearch-native-realm) and log in as the `elastic` superuser on Kibana.
+
+Problems? If you're an Elastic Enterprise Search engineer, please reach out to @elastic/enterprise-search-frontend for questions or our in-depth Getting Started developer guide.
 
```

Comment:
It's my first time reading this readme. Perhaps consider adding a link to [Contributing](https://www.elastic.co/guide/en/kibana/master/contributing.html) or the [Developer guide](https://www.elastic.co/guide/en/kibana/master/development.html)

Reply:
Can you clarify what you mean Rich? Are you saying we should A) add a link to this README to the pages you linked above, or B) link to the contributing/developer guide from this README?

If B), there are links to both in our Getting Started guide. There's a separate thread going on about how open to the community we want that guide to be and how open to the community generally our plugin should feel, but this README at least points a breadcrumb there for the primary engineers we expect to be contributing to the plugin. For non-Enterprise Search engineers (i.e., other Elastic engineers or community contributors), I fully expect them already to be familiar with Kibana's dev/contributing guides: it's linked in the main Kibana-level README and they have to sign a contributor agreement to merge a PR.

Reply:
> there are links to both in our Getting Started guide.

Thanks, then no need to duplicate the links here.

> super interesting q on how we want to consider/handle open source contributions in the future though - will bring this up in the larger eng channel if that's cool!

👍

Reply:
Sorry, I shouldn't answer comments first thing in the morning with not enough coffee ☕ I totally forgot to comment that it would be helpful (IMO) to Ent Search engineers to actually link to the main Kibana-level README and make sure they've skimmed that and its links. I'll add that and that should work well in lieu of the contributing/developer guide links!

Reply:
Addressed Rich's comment: https://github.com/elastic/kibana/pull/105151/commits/9fa95787cb9d33943e908b49d0b66059598dbf99

Adding this made me realize Kibana's README needs some updates lol, since 2 of those links are dead. I pinged the team on Slack.